src/IssueViz/IssueVisualizationControl/ViewModels/IssueVisualizationViewModel.cs

```diff
@@ -83,16 +83,15 @@ internal sealed class IssueVisualizationViewModel : IIssueVisualizationViewModel
                 selectionService.SelectedLocation);
         }
 
-        public int LineNumber
+        public int? LineNumber
         {
             get
             {
                 var issueSpan = CurrentIssue?.Span;
 
                 if (issueSpan == null || issueSpan.Value.IsEmpty)
                 {
-                    selectionService.SelectedIssue = null;
-                    return 0;
+                    return null;
```

Comment:
As we discussed offline, this is part of the clean-up as preparation for the new selection service.
I opened #2098 to handle the previous behavior.